e164-world.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Why do we need these?

Reply:
they're in the original data set, and not in the north american data set coming from NANP. we don't actually need them, but would rather keep it the same than change it.

Reply:
Is this file generated? If so should we add a comment at the top when it's generated to say don't edit directly?

Reply:
It's not, I split it out manually from the original data.  Not sure what to add as a comment.

Reply:
also no comments in json, whoops.